weaver/core/drivers/fabric-driver/server/satp.ts

```diff
@@ -0,0 +1,363 @@
+import satp_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_pb';
+import satp_grpc_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_grpc_pb';
+import driverPb from '@hyperledger/cacti-weaver-protos-js/driver/driver_pb';
+import logger from './logger';
+import { credentials } from '@grpc/grpc-js';
+import { SatpAssetManager, HashFunctions } from '@hyperledger/cacti-weaver-sdk-fabric'
+import fs from 'fs';
+import path from 'path';
+import { Gateway, Network, Wallets, Contract, X509Identity } from 'fabric-network'
+import { getNetworkGateway } from "./fabric-code";
+import { getDriverKeyCert } from './walletSetup';
+
+const DB_NAME: string = "driverdb";
+const DRIVER_ERROR_CONSTANTS = JSON.parse(
+    fs.readFileSync(
+        path.resolve(__dirname, '../constants/driver-error-constants.json'),
+    ).toString(),
+);
+
+const getWallet = (walletPath: string) => {
+    return Wallets.newFileSystemWallet(walletPath);
+};
+
+async function performLockHelper(
+    performLockRequest: driverPb.PerformLockRequest,
+    networkName: string
+): Promise<any> {
+
+    // TODO: remove the hardcoded values 
+    let performLockRequest2 = {};
+    performLockRequest2['target-network'] = 'network1';
+    performLockRequest2['hashBase64'] = 'ivHErp1x4bJDKuRo6L5bApO/DdoyD/dG0mAZrzLZEIs=';
+    performLockRequest2['timeout-duration'] = parseInt('3600');
+    performLockRequest2['locker'] = 'alice';
+    performLockRequest2['recipient'] = 'bob';
+    performLockRequest2['lockerWalletPath'] = '../wallet-network1/alice.id';
+    performLockRequest2['recipientWalletPath'] = '../wallet-network1/bob.id';
+    performLockRequest2['param'] = 'bond01:a05';
+    performLockRequest2['channel'] = 'mychannel';
+    performLockRequest2['chaincode-id'] = 'satpsimpleasset';
+
+    // Locker and recipient
+    const locker = performLockRequest2['locker'];
+    const recipient = performLockRequest2['recipient'];
+    const lockerWalletPath = performLockRequest2['lockerWalletPath'];
+    const recipientWalletPath = performLockRequest2['recipientWalletPath'];
+    let hashFn = performLockRequest2['hash_fn'];
+    let hashBase64 = performLockRequest2['hashBase64'];
+    const targetNetwork = performLockRequest2['target-network'];
+    const channel = performLockRequest2['channel'];
+    const chaincodeId = performLockRequest2['chaincode-id'];
+
+    // Hash
+    let hash: HashFunctions.Hash
+    if (hashFn == 'SHA512') {
```

Comment:
Use strict equality `===`, also this fragment can be simplified with use of ternary operator

Reply:
@outsidethecode Please fix this.

Reply:
Fixed.